option optimize_for = LITE_RUNTIME;

message CrusadeFortressBattleRecordRequest
{
 required uint32 id = 1;
}
